en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+# Cocos Creator v2.1.0 Release Notes
+
+__Cocos Creator__ __v2.1__, adds support for 3D features such as: **3D model rendering**, **3D Camera**, **3D skeletal animation**, and **3D point selection**. The __Cocos Creator__ editor now natively supports parsing 3D model files in **FBX format**. It's that simple, no additional workflow is required!
+
+In the previous version, v2.0, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer to officially introduce 3D support. The addition of 3D features can greatly enrich the expressiveness of 2D games, reduce the resource overhead of 2D games, and bring more imagination and ideas to expand games.
+However, it should be emphasized that the target users of __Cocos Creator__ is still 2D game development, and the **2D game is still the core** service of __Cocos Creator 2.x__. We will continue to optimize the development experience and basic performance of 2D games, and gradually increase the 3D functions suitable for integration into 2D games. Therefore, __Cocos Creator__ does not allow a pure 3D game, which is the goal of __Cocos3D__. __Cocos Creator's__ development path is closer to 2.5D.
+
+This is not the first time __Cocos__ has upgraded the engine from 2D to 3D. A few years ago, __Cocos2d-x__ expanded to the 3D space and accumulated a lot of experience. It also produced a very good game concept like *Sword and Home*. However, at that time, 3D functions were piled up on a very heavy 2D renderer, and there was no editor support. It was not widely recognized by the market. This time, __Cocos Creator__ is based on __Cocos3D's__ 3D renderer, which has a lot of 3D stability and guarantees a seamless editor experience.
+
+## Follow-up Version Notice
+
+**v2.0 will continue to be maintained, native platform performance will be optimized as soon as possible**
+
+__v2.0__ and __v2.1__ will grow together over a period of time.__v2.0__ will continue to be maintained, adding more performance optimizations and bug fixes. We will soon be beta testing v2.0.7. In v2.0.7 the rendering performance of the engine has been optimized, especially **Spine** and **DragonBones**. Also, the performance has been greatly improved.
+
+**v2.1 iterations will be more aggressive**
+
+Compared with __v2.0__, the upgrade cycle of __v2.1__ will mainly focus on 3D features. We recommend upgrading new projects with 3D requirements to this version, and establishing contact with us on our forums. This allows us and our 3D partners to provide support.Since v2.1 is a very new version, we will speed up the iteration and try to meet the needs of developers. In particular, **currently v2.1.0 version still does not support 3D scene editing, only the 3D parameters can be set in the property inspector**. We will be releasing **v2.1.1** in the near future to support 3D scene editing, material systems and optimizing the 3D development process.
```

Comment:
1. ...provide support.Since v2.1... -> ...provide support. Since v2.1 is a....Missing spaces at the end of the sentence
2. property inspector -> Properties
3. ...in the near future to support 3D scene editing, material...development process. -> ...in the near future, which is used to support 3D scene editing and material systems to optimize the 3D development process.

Reply:
1. Thanks. I had noticed.

2. Thank you.

3. __which is used__ doesn't make sense. I changed to this:  We will be releasing **v2.1.1** in the near future which will support *3D scene editing*, *material systems* and *optimizing the 3D development process*.